api/src/main/java/org/openmrs/module/idgen/IdgenModuleActivator.java

```diff
@@ -43,4 +46,25 @@ public void stopped() {
 	public void setDaemonToken(DaemonToken token) {
 		IdgenTask.setDaemonToken(token);
 	}
+	
+	/**
+	 * Creates the location attribute type for location prefixes consumed by prefix providers.
+	 * 
+	 * @see {@link PrefixProvider}
+	 * 
+	 * @return The created location attribute type.
+	 */
+	public LocationAttributeType createLocationPrefixAttributeType() {
```

Comment:
See above, I don't think this module should create a location attribute type directly